cmd/telemetrygen/internal/traces/config.go

```diff
@@ -31,6 +32,7 @@ func (c *Config) Flags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.HTTPPath, "otlp-http-url-path", "/v1/traces", "Which URL path to write to")
 
 	fs.IntVar(&c.NumTraces, "traces", 1, "Number of traces to generate in each worker (ignored if duration is provided)")
+	fs.IntVar(&c.NumSpans, "spans", 1, "Number of spans to generate for each trace")
```

Comment:
This is specifically the number of child spans, please call this out in the name/help docs, otherwise I would imagine this being confusing (as there would be 2 spans generated when the number of spans is 1)

Reply:
Thanks a bunch for the feedback. Renamed `spans` to `child-spans` and other related variable names.